translations/README.russian.md

```diff
@@ -0,0 +1,199 @@
+# Hasura GraphQL Engine
+
+[![Docs](https://img.shields.io/badge/docs-v1.0-brightgreen.svg?style=flat)](https://docs.hasura.io)
+[![CircleCI](https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)](https://circleci.com/gh/hasura/graphql-engine)
+
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine - это сверхбыстрый графический сервер позволяющий создавать **мгновенно GraphQL API на основе Postgres**, с [**webhook triggers**](event-triggers.md), и [**remote schemas**](remote-schemas.md) для бизнес-логики.
+
+Hasura помогает стройть GraphQL приложения при поддержке Postgres или постепенно перейти на GraphQL для уже существующих приложений, использующих Postgres.
+
+Подробнее на [hasura.io](https://hasura.io) и [docs](https://docs.hasura.io).
+
+------------------
+
+![Hasura GraphQL Engine Demo](assets/demo.gif)

```

Comment:
This doesn't show up correctly: 

![image](https://user-images.githubusercontent.com/5722022/67632424-62a74d80-f8a3-11e9-98ad-2c47248ea5e0.png)